### ISSUE TITLE
No power / battery settings for CM4

### ISSUE THREAD

**Issue by scyto** [clockworkpi/uConsole#11]
I expected to be able to set power and battery settings in the desktop on the CM4.

Was surprised when i couldn't.  Does the CM4 module support sleep states?

**Comment by neggles** [comment]
CM4/Pi 4 have absolutely no support whatsoever for any kind of sleep state, no 😢

**Comment by lambtor** [comment]
is there any way to configure the battery level icon by telling it what capacity your batteries claim to have?

**Comment by neggles** [comment]
> is there any way to configure the battery level icon by telling it what capacity your batteries claim to have?

Yes, there is - but you have to modify the kernel/devicetree binaries to do it. I did this over in [my kernel fork](https://github.com/neggles/linux-rpi/commits/uconsole-6.6.y/) where I have a better version of the LCD driver and a few other things, too 😄
